README.md

```diff
@@ -93,11 +95,11 @@ az repos import create --git-source-url $REPOSITORY_URL --repository $REPOSITORY
 
 ### 2. Creating or reusing a service principal
 
-Azure service principal is an identity created for use with applications, hosted services, and automated tools to access Azure resources. This access is restricted by the roles assigned to the service principal, giving you control over which resources can be accessed and at which level. 

```

Comment:
Should we use `az devops` to create the service connection?
https://docs.microsoft.com/en-us/cli/azure/ext/azure-devops/devops/service-endpoint/azurerm?view=azure-cli-latest

If so, should we replace this section and give instructions on how to do it?

Reply:
its a good idea. I will make that change soon.

Reply:
We can update this in another PR if you prefer, no worries.